mocks.go

```diff
@@ -0,0 +1,11 @@
+package main
```

Comment:
it makes the most sense to move this to _./utils/docker/_ path

Reply:
We could, but I took a different approach - like with [tools.go](https://github.com/mesg-foundation/core/blob/master/tools.go) - put every interface (from vendor) that we want to mock in one palce and not spread over different directories. Maybe it should be good to add comment why we have it here.

Reply:
```suggestion
package mocks
```

Why not naming it mocks in this case? Like tools?